include/vara/Feature/ConstraintParser.h

```diff
@@ -497,6 +499,30 @@ class ConstraintParser {
     }
   }
 
+  static std::unique_ptr<Constraint> parseUnsigned(llvm::StringRef Str) {
+    if (Str.contains_lower('e')) {
```

Comment:
Did they rename the method? In the StringRef API I only find `contains_insensitive` and not `contains_lower`

Reply:
This method was only renamed six months ago (https://github.com/llvm/llvm-project/commit/3eed57e7ef7da5eda765ccc19fd26fb8dfcd8d41), therefore i use the old name with our LLVM version.
This will probably break at some point, as we update the framework.

Reply:
:+1: I thought something like this :)